src/components/AddProposal/AddProposal.component.tsx

```diff
@@ -0,0 +1,186 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { useFieldArray, useForm } from 'react-hook-form';
+import { AppDialog } from '../AppDialog/AppDialog.component';
+import { Button } from '../Button/Button.component';
+import { ControlledDropdown } from '../Dropdown/Dropdown.controlled';
+import { ControlledInput } from '../TextInput/TextInput.controlled';
+import { AddProposalFields, AddProposalProps } from './AddProposal.types';
+import { AddProposalInputs } from './AddProposalFields';
+
+const AP_DD_OPTIONS = [
```

Comment:
please add `AddProposal.container` file that will be responsible of providing the submit function and all other blockchain operations